### PR TITLE
Use own Rubocop rules

### PR DESCRIPTION
**Pull request by janschill**
Clean code that is formatted to a standard is useful
to not lose sanity when working with mulitple people
on a project.

### CODE REVIEW

File: api/minitwit_sim_api.rb [hunk 1]
```diff
@@ -83,7 +80,7 @@ class SimAPI < Roda
         end
 
         r.is String do |username|
-          username = URI.unescape(username)
+          username = CGI.unescape(username)
```
Comment (Fr3d-) [inline api/minitwit_sim_api.rb]: Is this related to the robucop rule change?

Comment (janschill) [reply api/minitwit_sim_api.rb]: Yes, it was brought to my attention through Rubocop that URI is deprecated.
